build.gradle

```diff
@@ -106,15 +106,15 @@ allprojects {
     vendorName = 'SnappyData, Inc.'
     scalaBinaryVersion = '2.11'
     scalaVersion = scalaBinaryVersion + '.8'
-    sparkVersion = '2.1.1'
-    snappySparkVersion = '2.1.1.1'
+    sparkVersion = '2.3.0'
+    snappySparkVersion = '2.3.0'
```

Comment:
Generally the ordinal version we use (snappySparkVersion) is one ahead of Spark version; like 2.3.0.1

Reply:
Ah right, thanks